src/encoded/static/components/region_search.js

```diff
@@ -83,53 +85,43 @@ class AdvSearch extends React.Component {
         }
+        this.setState({ searchInput: replaceNewline(exampleString) });
+    }
 
-        let exampleString = "";
-        if (e.target.id === "example-snps") {
-            exampleString = "rs3768324\nrs75982468\nrs10905307\nrs10823321\nrs7745856";
-        } else if (e.target.id === "example-coordinates") {
-            exampleString = "chr11:62607065-62607067\nchr10:5894500-5894501\nchr10:11741181-11741181\nchr1:39492463-39492463\nchr6:10695158-10695160";
-        } else {
-            exampleString = "rs3768324\nrs75982468\nrs10905307\nrs10823321\nrs7745856";
-        }
-        document.getElementById("multiple-entry-input").value = replaceNewline(exampleString);
+    handleOnFocus() {
+        this.context.navigate(this.context.location_href);
     }
 
     render() {
         const context = this.props.context;
-        const id = url.parse(this.context.location_href, true);
-        const region = id.query.region || '';
         const searchBase = url.parse(this.context.location_href).search || '';
 
         return (
             <Panel>
                 <PanelBody>
                     <form id="panel1" className="adv-search-form" autoComplete="off" aria-labelledby="tab1" onSubmit={this.handleOnFocus} >
                         <div className="form-group">
-                            <label htmlFor="annotation"><i className="icon icon-search"></i>Search by dbSNP ID or coordinate range (hg19)</label>
+                            <label htmlFor="annotation">
+                                <i className="icon icon-search" />Search by dbSNP ID or coordinate range (hg19)
+                            </label>
                             <div className="input-group input-group-region-input">
-                                <textarea className="multiple-entry-input" id="multiple-entry-input" placeholder="Enter search parameters here." onChange={this.handleChange} name="regions">
-                                </textarea>
+                                <textarea className="multiple-entry-input" id="multiple-entry-input" placeholder="Enter search parameters here." onChange={this.handleChange} name="regions" value={this.state.searchInput} />
 
-                                <p className="example-inputs" onClick={this.handleExamples}>Click for example entry: <span className="example-input" id="example-snps">multiple dbSNPs</span> or <span className="example-input" id="example-coordinates">coordinates ranges</span></p>
+                                <p className="example-inputs">
+                                    Click for example entry: <span className="example-input" id="example-snps" onClick={e => this.handleExamples(e)} onKeyDown={this.handleExamples} role="button" tabIndex={0}>multiple dbSNPs</span> or <span className="example-input" id="example-coordinates" onClick={e => this.handleExamples(e)} onKeyDown={this.handleExamples} role="button" tabIndex={0}>coordinates ranges</span>
```

Comment:
I tried to write this so that each example was a component, and it works, except it also submits the input by triggering handleOnFocus, which is confusing and not what we want. Can you take a look at that?